frontend/components/commons/header/header-feedback-task/HeaderFeedbackTask.component.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
Is it not possible to use the standard header for all tasks? In case you need a specific one, better not to use the commons folder (for the components common to all tasks) and put it inside a folder (as a text classifier).

Reply:
For later yes. But for the very first fast iterations, since we need to point to a new page with a new route it's better to use another header I think and then do the refactoring for homogenisation